### PR TITLE
Same values for follower list and reward table

### PR DESCRIPTION
**Pull request by keikari**
Set values that match the values in "Followers reward" table into the "Coveted Creator" section of list

### Fixes #

### CODE REVIEW

File: content/faq/rewards.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ Rewards allow LBRY users to earn LBC, the token that powers the LBRY network. Yo
 
 **Please note:** Reward amounts are subject to change at any time by LBRY Inc.
 
-![rewardslist](https://spee.ch/rewardslist.png)
```
Comment (keikari) [inline content/faq/rewards.md:10]: This is something that my browser did.

**Comment by tzarebczan** [comment]
Thanks again - we'll most likely be adjusting this soon again and I don't think the values need to match exactly. Maybe you can alter a screenshot that says "LBC values may not match, see table for actual reward amount". Can we show you some [appreciation](https://lbry.com/faq/appreciation)?

**Comment by keikari** [comment]
Thanks for appreciation offer, but I have gotten much more than my content deserves(some gameplay videos to test lbry) from the view rewards, so that's enough appreciation :D 
Should I close this pull request then, I don't really know how git works.

**Comment by tzarebczan** [comment]
You can just edit the existing file and let me know.

**Comment by tzarebczan** [comment]
Closing for now, you can submit a new PR :)

**Pull request closed without merging**